docs/device_report.md

```diff
@@ -152,9 +157,89 @@ RESULT skip protocol.bacnet.pic Bacnet device not found... Pics check cannot be
 ## Module tls
 
 ```
-IOException unable to connect to server.
-RESULT skip security.tls.v3
-RESULT skip security.x509
+Cipher:
```

Comment:
this should be a skip I believe, try not to boot up the nginx server in the start_faux